### DISCUSSION TITLE
Icon

### DISCUSSION THREAD

**Discussion by God-damnit-all** [Sophia-Community/SophiApp#14] [Ideas]
I processed the [image that Sophia Script for Windows uses](https://github.com/farag2/Sophia-Script-for-Windows/blob/master/img/Sophia.png) into a mipmapped icon, please use it:

[Sophia.ico.zip](https://github.com/Sophia-Community/SophiApp/files/7723575/Sophia.ico.zip)

**Comment by farag2** [comment]
I'm going to order a logo this month. But we'll check it out. Thanks. :)
